deps/build.jl

```diff
@@ -4,25 +4,25 @@ using BinDeps
 
 blasvendor = Base.BLAS.vendor()
 
+libs = Dict(:direct=>"libscsdir", :indirect=>"libscsindir")
 
-if (is_apple() ? (blasvendor == :openblas64) : false)
-    aliases = ["libscsdir64"]
-else
-    aliases = ["libscsdir"]
+if is_apple() && (blasvendor == :openblas64)
+    libs = Dict(k => v*"64" for (k,v) in libs)
 end
 
-scs = library_dependency("scs", aliases=aliases)
+direct = library_dependency("direct", aliases=[libs[:direct]])
+indirect = library_dependency("indirect", aliases=[libs[:indirect]])
 
 if is_apple()
     using Homebrew
-    provides(Homebrew.HB, "scs", scs, os = :Darwin)
+    provides(Homebrew.HB, [libs[:direct], libs[:indirect]], [direct, indirect], os = :Darwin)
```

Comment:
Revert this change. Homebrew will still provide a single package called "scs".

Reply:
Do You mean
```julia
provides(Homebrew.HB, "scs", [direct, indirect], os = :Darwin)
```
so that Homebrew "scs" package will provide both direct and indirect lib?

Reply:
Yes. But probably best to comment out this code also and leave a TODO because it doesn't work.